packages/@uppy/companion/src/server/provider/zoom/index.js

```diff
@@ -225,7 +226,7 @@ class Zoom extends Provider {
       items: [],
       username: adapter.getUserEmail(userResponse)
     }
-    const items = results.meetings || results.recording_files
+    const items = results.meetings || results.recording_files.filter(file => file.file_type !== 'TIMELINE')
```

Comment:
timeline files are json files that are intended for programmatic use, so we should omit them from the user's view (the same way that zoom omits these files from the their recording files list in zoom.us/recordings).